sdk/core/System.ClientModel/src/Options/ClientPipelineOptions.cs

```diff
@@ -117,6 +118,21 @@ public ClientLoggingOptions? ClientLoggingOptions
         }
     }
 
+    /// <summary>
+    /// Whether distributed tracing should be enabled. If <c>null</c>, this
+    /// value will be treated as <c>true</c>.
```

Comment:
(Assuming the default is null)
```suggestion
    /// Gets or sets whether distributed tracing should be enabled. If <c>null</c>, this
    /// value will be treated as <c>true</c>. The default is <c>null</c>.
```